api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClientImpl.java

```diff
@@ -143,10 +159,13 @@ public void createRuntime(
     leonardoRetryHandler.run(
         (context) -> {
           runtimesApi.createRuntime(
-              googleProject,
-              runtimeName,
+              runtime.getGoogleProject(),
```

Comment:
If we're getting the project value here, we should also be rejecting requests where this does not matching the URL workspaceaNamespace. However, it may be cleaner just to ignore the value that's in the runtime payload.